### ISSUE TITLE
error in RNA report on test data: cannot open the connection

### ISSUE THREAD

**Issue by slsevilla** [CCBR/RENEE#82]
Ran RENEE v2.5.9 from the GUI on the test data:

Process errored here: 
```
Error in file(con, "w") : cannot open the connection
Calls: <Anonymous> -> <Anonymous> -> write_utf8 -> writeLines -> file
In addition: There were 16 warnings (use warnings() to see them)
Execution halted
[Fri Jan  5 12:31:59 2024]
Error in rule rna_report:
    jobid: 0
    input: /data/sevillas2/RENEE/2.5.9_v1/DEG_ALL/RSEM.genes.expected_counts.all_samples.reformatted.tsv, /data/sevillas2/RENEE/2.5.9_v1/DEG_ALL/combined_TIN.tsv, /data/sevillas2/RENEE/2.5.9_v1/Reports/multiqc_matrix.tsv
    output: /data/sevillas2/RENEE/2.5.9_v1/Reports/RNA_Report.html
    shell:
        
    # Avoids inheriting $R_LIBS_SITE
    # from local env variables
    R_LIBS_SITE=/usr/local/lib/R/site-library
    # Generate RNA QC Dashboard
    workflow/scripts/rNA.R         -m workflow/scripts/rNA_flowcells.Rmd         -r /data/sevillas2/RENEE/2.5.9_v1/DEG_ALL/RSEM.genes.expected_counts.all_samples.reformatted.tsv         -t /data/sevillas2/RENEE/2.5.9_v1/DEG_ALL/combined_TIN.tsv         -q /data/sevillas2/RENEE/2.5.9_v1/Reports/multiqc_matrix.tsv         -o /data/sevillas2/RENEE/2.5.9_v1/Reports         -f RNA_Report.html
```

Process completed with all steps except for writing the output report:
```
...
label: loadings-heatmap (with options) 
List of 2
 $ dpi       : num 300
 $ fig.height: num 6

  |..................................................................... |  98%
   inline R code fragments

  |......................................................................| 100%
label: session-info
```

**Comment by slsevilla** [comment]
Tested in R-Studio and was able to knit the report without an error.

**Comment by slsevilla** [comment]
Tested on the command line with '--mode local' and got the same error listed above.

**Comment by kelly-sovacool** [comment]
Oddly, this fails for you, Vishal, and others, but works for me with v2.5.10.

**Comment by slsevilla** [comment]
Found a similar error: https://github.com/boxuancui/DataExplorer/issues/20

It appears because I don't have write permission to the R library location, that invocation of the rule fails to create the knit files. All of the fixes I've tried so far haven't worked though. I'm still only able to create it within R studio but cannot create it using Snakemake or just calling the R file directly.

**Comment by slsevilla** [comment]
The /output/dir/workflow directory is being set to 'read-only' by RENEE at some point. This is causing the creation of the 'knit.md' file to be unable to save and causing the pipeline to fail at this rule. I am going to test with a clean pull of the repo to ensure this fixes the issue

**Comment by kelly-sovacool** [comment]
We need to add a line to change the permissions of the output Rmd subdir after copying it

**Comment by slsevilla** [comment]
Running test to ensure this is working with current main branch
```
./renee run --input .tests/*.gz --output /data/sevillas2/renee_2.5.11/ --genome hg38_30 --mode slurm
```

**Comment by kelly-sovacool** [comment]
> Running test to ensure this is working with current main branch
> 
> ```
> ./renee run --input .tests/*.gz --output /data/sevillas2/renee_2.5.11/ --genome hg38_30 --mode slurm
> ```

Will this work on the main branch when we haven't merged #91 yet?
